cmd/sink/logs/logs.go

```diff
@@ -0,0 +1,81 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package logs
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/kubearchive/kubearchive/cmd/sink/k8s"
+	ocel "github.com/kubearchive/kubearchive/pkg/cel"
+	"github.com/kubearchive/kubearchive/pkg/log_urls"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	configCmName     = "kubearchive-config"
+	loggingCmKeyName = "logging-configmap-name"
+	logUrlKey        = "LOG_URL"
+	celPrefix        = "cel:"
+)
+
+func getKubeArchiveLoggingCm(ctx context.Context, kubeClient kubernetes.Interface) (*corev1.ConfigMap, error) {
+	kaConfigCm, err := kubeClient.CoreV1().ConfigMaps(k8s.KubeArchiveNamespace).Get(
+		ctx,
+		configCmName,
+		metav1.GetOptions{},
+	)
+	if err != nil {
+		return nil, fmt.Errorf("Could not get ConfigMap: %w", err)
+	}
+	loggingCmName, exists := kaConfigCm.Data[loggingCmKeyName]
+	if !exists {
+		return nil, fmt.Errorf("No logging ConfigMap was provided")
```

Comment:
Why is this an error?  Logging is not required.

Reply:
`getKubeArchiveLoggingCm` no longer returns errors. If the an issue is encountered it returns a `ConfigMap` with an empty `Data` map